include/rocksdb/options.h

```diff
@@ -199,13 +199,18 @@ struct ColumnFamilyOptions : public AdvancedColumnFamilyOptions {
   // incompressible, the kSnappyCompression implementation will
   // efficiently detect that and will switch to uncompressed mode.
   //
+  // In kLZ4 compression, the `acceleration` parameter is configurable by
```

Comment:
Why do we use negated value?

Reply:
I am used to decreasing `CompressionOptions::level` to make compression faster at the expense of worse compression ratio. `acceleration`, however, needs to be increased to get that tradeoff

Reply:
I see. Maybe we can add that to comment about smaller value of level means faster for LZ4 and that user should set it to negative values if needed.

Reply:
OK, tried to do that. The comments are mostly in advanced_options.h. Last week I didn't even realize the API doc we are commenting on here is not for `CompressionOptions::level`.